views/Create Account.txt

```diff
@@ -0,0 +1,8 @@
+<header>
+
+<footer>
+( Entering 'cancel' will abort the account build )
+
+<@builder.issues>
+Please Enter <@builder.current_field>
+-$ <prompt>
```

Comment:
Lets make our prompts uniform across the application. So either change the '$' to '>' or change all '>' prompts to the '$' prompt.

Reply:
ill change it to >. uniformity makes sense

Reply:
using the <prompt> command now